src/Schematron.Tests/ValidatorTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using System.Xml.XPath;
+using System.Xml.Serialization;
+using System.Text;
+using System.Xml.Schema;
+using Xunit;
+
+namespace Schematron.Tests
+{    
+    public class ValidatorTests
+    {
+        const string XsdLocation = "./Content/po-schema.xsd";
+        const string XsdWithPartialSchemaLocation = "./Content/po-schema-with-schema-import.xsd";
+        const string XmlContentLocation = "./Content/po-instance.xml";
+        const string TargetNamespace = "http://example.com/po-schematron";
+
+        [Fact]
+        public void NewAddSchemaSignatureShouldNotBreakCode()
```

Comment:
That test is a little bit lame in fact (starting with the name). Since the newly added .AddSchema() signature relies on a slightly different implementation just wanted to make sure that both implementation had the same result in terms of validation outpu (not only schematron related but xml schema as well).